### PR TITLE
[QC-977] Select automatically the correct params 

### PR DESCRIPTION
**Pull request by Barthelemy**
based on the current activity.

### CODE REVIEW

**Comment by Barthelemy** [comment]
doc and test coming

**Comment by Barthelemy** [comment]
ready

File: Framework/src/CustomParameters.cxx [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // or submit itself to any jurisdiction.
 
 #include "QualityControl/CustomParameters.h"
-
+#include "DataFormatsParameters/ECSDataAdapters.h"
```
Comment (knopers8) [inline Framework/src/CustomParameters.cxx:13]:
```suggestion
#include <DataFormatsParameters/ECSDataAdapters.h>
```

File: Framework/src/CustomParameters.cxx [hunk 2]
```diff
@@ -66,6 +66,17 @@ std::optional<std::string> CustomParameters::atOptional(const std::string& key,
   }
 }
 
+std::optional<std::string> CustomParameters::atOptional(const std::string& key, const Activity& activity) const
+{
+  // Get the proper parameter for the given activity
+  const int runType = activity.mType; // get the type for this run
+  // convert it to a string (via a string_view as this is what we get from O2)
+  const std::string_view runTypeStringView = o2::parameters::GRPECS::RunTypeNames[runType];
+  const std::string runTypeString = { runTypeStringView.begin(), runTypeStringView.end() };
```
Comment (knopers8) [inline Framework/src/CustomParameters.cxx:75]:
```suggestion
  const std::string runTypeString{ runTypeStringView };
```
doesn't this work? a bit simpler.

**Review by knopers8 — APPROVED** [review]
i have two small correction proposals, but it looks good as it is

**Comment by Barthelemy** [comment]
thank you for the suggestions, I committed them.

**Pull request merged**